corehq/apps/data_interfaces/forms.py

```diff
@@ -86,11 +86,11 @@ def validate_case_property_name(value, allow_parent_case_references=True):
     return value
 
 
-def hidden_bound_field(field_name):
+def hidden_bound_field(field_name, data_value):
     return Field(
         field_name,
         type='hidden',
-        data_bind='value: %s' % field_name,
+        data_bind='value: %s' % data_value,
```

Comment:
nit: can consider renaming it to `field_value_attr` or something better
@orangejenny might have better suggestion